### ISSUE TITLE
Increase possible memory size for name/clan within demos

### ISSUE THREAD

**Issue by oy** [teeworlds/teeworlds#2762]

**Comment by fokkonaut** [comment]
You labeled this as 0.8. Can we get rid of those demo messages maybe for 0.8? Also, generally talking about snapshots, I think we should make a clear plan for them for a new major update, because stuff like names, clans and skins really should be placed in snapshots like before.

**Comment by oy** [comment]
No, it's much more flexible that way, they shouldn't be part of snapshots again and wouldn't fit anyway.

**Comment by ChillerDragon** [comment]
Also skin part names containing unicode can exceed the 6 integers and will be truncated in demos. See https://github.com/ddnet/ddnet/issues/9119
